codegen/src/main/scala/caliban/codegen/ClientWriter.scala

```diff
@@ -249,9 +249,8 @@ object ClientWriter {
         )
       }
     val args = field.args match {
-      case Nil => ""
-      case list =>
-        s"(${list.map(arg => s"${safeName(arg.name)}: ${writeType(arg.ofType)}").mkString(", ")})"
+      case Nil  => ""
+      case list => s"(${writeArgumentFields(list)})"
```

Comment:
You can remove the quotes now.

Reply:
I need to add parens in this case, I believe. Is that what you mean?

Reply:
Ah sorry i was not seeing properly. All good!